src/passes/Print.cpp

```diff
@@ -599,6 +603,7 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
   }
   void visitFunction(Function *curr) {
     currFunction = curr;
+    lastPrintedLocation = {0, 0, 0};
```

Comment:
i think space after `{` and before `}` is what we have more commonly in the codebase.